apply/builder.go

```diff
@@ -120,6 +144,17 @@ func (ab *Builder) ExecuteApply() error {
 				return fmt.Errorf("resource collection failed for file %s: %w", filename, err)
 			}
 
+			if ab.applyFilter != nil {
+				ok, err := ab.applyFilter.Predicate(yamlDoc)
+				if err != nil {
+					return fmt.Errorf("filtering resource failed for file %s: %w", filename, err)
+				}
+
+				if !ok {
```

Comment:
If the filter returns true the resource will be applied. That is not natural filtering.

Reply:
It is natural filtering as I need to state which elements are going to be applied.